specification/imagebuilder/resource-manager/Microsoft.VirtualMachineImages/preview/2018-02-01-preview/imagebuilder.json

```diff
@@ -0,0 +1,799 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "VirtualMachineImageTemplate",
+    "description": "Virtual Machine Image Template",
+    "version": "2018-02-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.VirtualMachineImages/imageTemplates/{imageTemplateName}": {
+      "put": {
+        "tags": [
+          "VirtualMachineImageTemplate"
+        ],
+        "operationId": "VirtualMachineImageTemplate_CreateOrUpdate",
+        "description": "Create or Update a Virtual Machine Image Template",
+        "parameters": [
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ImageTemplate"
+            },
+            "description": "Parameters supplied to the Create Image Template"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ImageTemplateNameParameter"
+          }
+        ],
+        "responses": {
```

Comment:
"default" response which captures all the error responses is missing. The schema complies with the error contract defined in ARM RPC. Reference batch RP swagger.